ts/src/helpers.cpp

```diff
@@ -279,12 +279,17 @@ std::map<std::string, FREQ_SETTINGS> helpers::parseFrequencies(const std::string
     std::vector<std::string> v = split(sub, ',');
     for (const std::string& xs : v) {
         std::vector<std::string> parts = split(xs.substr(1, xs.length() - 2), '|');
-        if (parts.size() == 3 || parts.size() == 4) {
+        if (parts.size() == 3 || parts.size() == 4 || parts.size() == 5) {
```

Comment:
We are in Beta phase right now. We don't need to care about being compatible to older beta builds.
I regularly break backwards compat.